app/controllers/api/mixins/tags.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module Mixins
+    module Tags
+      def tag_specified(id, data)
+        if id.to_i > 0
+          klass  = collection_class(:tags)
+          tagobj = klass.find(id)
+          return tag_path_to_spec(tagobj.name).merge(:id => tagobj.id)
+        end
+
+        parse_tag(data)
+      end
+
+      def parse_tag(data)
+        return {} if data.blank?
+
+        category = data["category"]
+        name     = data["name"]
+        return {:category => category, :name => name} if category && name
+        return tag_path_to_spec(name) if name && name[0] == '/'
```

Comment:
Side note [was like this, out of scope to this PR]: this differs from documentation.
http://manageiq.org/docs/reference/fine/api/reference/tagging#tag-specification says you give full path in "path" key.  This code takes full path in "name" (if "category" not given).
Dunno if right solution is to fix doc or code...